pos_order_return/models/pos_order.py

```diff
@@ -48,11 +48,10 @@ def _blank_refund(self, res):
 
     def _prepare_invoice(self):
         res = super(PosOrder, self)._prepare_invoice()
-        if not self.returned_order_id:
+        if not self.returned_order_id or not self.returned_order_id.invoice_id:
```

Comment:
Isn't the same to put `if not self.returned_order_id.invoice_id:` directly?

Reply:
Sure :)